style.css

```diff
@@ -218,3 +218,340 @@ body {
     margin-bottom: 20px;
   }
 }
+
+/* ///// */
+
+.container {
+  width: 90%;
+  margin: 60px auto;
+}
+
+.grid-wrapper {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(400px, auto));
+  grid-gap: 50px;
+}
+
+.grid-box {
+  padding: 15px;
+  height: 672px;
+}
+
+/* TESTIMONIALS */
+
+.works {
+  margin-top: 50px;
+  background-color: var(--light-pink);
+}
+
+.works .grid-wrapper {
+  grid-template-columns: repeat(auto-fit, minmax(250px, auto));
+}
+
+.works .grid-box {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  border-radius: 15px;
+  margin-top: 18px;
+  box-shadow: 5px 5px 30px rgba(56, 42, 42, 0.445);
+  position: relative;
+}
+
+.works .grid-box img {
+  margin-top: 10px;
+  align-self: center;
+  max-width: 95%;
+  height: 300px;
+}
+
+/* .testimonials .grid-box:hover img {
+  transform: scale(1.2);
+} */
+
+.content-box-works {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  text-align: center;
+}
+
+.title-grid-box {
+  margin-bottom: 15px;
+}
+
+.title-grid-box h2 {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.canopy-row h3 {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.canopy-row {
+  display: flex;
+  margin-bottom: 15px;
+}
+
+.canopy-row ul {
+  display: flex;
+  margin-left: 18px;
+}
+
+.canopy-row ul li {
+  display: flex;
+  margin-right: 8px;
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.content-box-works p {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  text-align: left;
+}
+
+.ul-buttons-lg {
+  display: flex;
+  list-style: none;
+  margin-bottom: 24px;
+}
+
+.li-buttons-lg {
+  margin-right: 10px;
+  width: fit-content;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.li-buttons-lg .buttons-lg {
+  padding: 4px 12px;
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+}
```

Comment:
- [ ] Make sure that you follow correctly the design from Figma to implement yours. 
### Your Design:
![image](https://user-images.githubusercontent.com/71874425/159017144-278d2e02-1963-4792-adb8-d0c077807e76.png)
### Figma design:
![image](https://user-images.githubusercontent.com/71874425/159017927-0903118a-2dd2-472f-8dad-2feb470c3061.png)

To point the Figma design, you can remove the border from those buttons by adding `border: none;` in the code above.